### ISSUE TITLE
Salvattore with Angular Js

### ISSUE THREAD

**Issue by panhna** [rnmp/salvattore#82]
The future feature it can support with Angular Js?

**Comment by nathankot** [comment]
quickly hacked up an angular module for salvattore: https://github.com/nathankot/angular-salvattore

salvattore should work with vanilla html, but if you want to wrap it around anything compiled by angular, you'll probably need a directive for it.

**Comment by richarddavenport** [comment]
@nathankot I looked at your directive, but didn't try it out. Does it actually work? I'd really like an Angular directive for salvattore!

**Comment by nathankot** [comment]
@richarddavenport yup :) but we never used salvattore for production in the end so there may be edge cases that haven't been smoothed out
